src/component/base/DraftEditorPlaceholder.react.js

```diff
@@ -21,7 +21,7 @@ const React = require('React');
 const cx = require('cx');
 
 type Props = {
-  accessibilityID: string,
+  accessibilityID: ?string,
```

Comment:
Rather than making the value nullable, make the property optional. If the property is defined, it must be a string.

Reply:
Looking better at this, I think it should be nullable. When the placeholder prop is not set, it's necessary to avoid to render the HTML attribute `aria-describedby` that would point to a non-existing element.